handlers/handlers.go

```diff
@@ -137,22 +121,16 @@ func edit(w http.ResponseWriter, req *http.Request, rendC RenderClient, siteDoma
 	cookies.SetPreferenceIsSet(w, siteDomain)
 	cookies.SetPolicy(w, cp, siteDomain)
 	isUpdated := true
-	err = getCookiePreferencePage(w, req, rendC, cp, isUpdated, lang)
+	getCookiePreferencePage(w, req, rendC, cp, isUpdated, lang)
 	if err != nil {
```

Comment:
Where is this 'err' value being populated to be being checked here ?

Reply:
Yep, oversight from me when updating the code here. This `err` check is not needed - will remove